CREDITS-LEVELS

```diff
@@ -45,7 +45,6 @@ Episode 4 - Double Impact
     E4M8: Launch Bay by Matt "RottKing" Cibulas and Ralph "Ralphis" Vickers
 
 
-
 --Phase 2--
 
 Cluster 1
```

Comment:
I know it's outside of the scope of your current change, but we have episode names, but not cluster names? Should we have cluster names? There is probably nothing for you to change here now, but maybe something we can talk about here or elsewhere.

Reply:
Cluster names have no use.

Reply:
They don't appear anywhere ingame and wouldn't be alluded to in the intermission texts, and in vanilla the sky doesn't even transition to let you know it's a new cluster. Frankly the idea of D2 clusters confused me when I first encountered them... which was maybe sometime around 2010? and I've been had Doom II since 1996.

I would be against even mentioning them, except that they help break up what would otherwise be an awfully long wall of undifferentiated list items.